data/shaders/planet.frag

```diff
@@ -434,7 +433,5 @@ void main()
     //apply white rimlight
     finalColor.xyz = clamp( finalColor.xyz + vec3(outgas), 0.0, 1.0);
 
-    FRAG_COLOR = vec4(linearToSRGB(finalColor.rgb), finalColor.a);
-    //to debug texture issues, uncomment and reload shader

```

Comment:
for the sake of less experienced GLSL devs, please leave such comments/alternative code paths visible.

Reply:
It would mean maintaining dead code. In particular, the following commented out line would be out of date if left as is.